Formula/muon.rb

```diff
@@ -0,0 +1,38 @@
+class Muon < Formula
+  desc "Meson-compatible build system"
+  homepage "https://muon.build"
+  url "https://git.sr.ht/~lattis/muon/archive/0.1.0.tar.gz"
+  sha256 "9d3825c2d562f8f8ad96d1f02167e89c4e84973decf205d127efd9293d7da35b"
+  license "GPL-3.0-only"
+  head "https://git.sr.ht/~lattis/muon", branch: "master"
+
+  depends_on "ninja"
+  depends_on "pkg-config"
+
+  def install
+    system "./bootstrap.sh", "build"
```

Comment:
Does this download anything?

Reply:
Maybe we can bootstrap with `meson` instead?

Reply:
No, this script just compiles a bootstrap version of muon by compiling a single C file that performs an include of all files. The source is simple and you can view it here: https://git.sr.ht/~lattis/muon/tree/master/item/bootstrap.sh